corehq/apps/accounting/tests/test_emails.py

```diff
@@ -0,0 +1,78 @@
+from unittest.mock import patch
+
+from corehq.apps.accounting.emails import (
+    _get_reminder_email_contacts,
+)
+from corehq.apps.accounting.models import WebUser
+from corehq.apps.accounting.tests.base_tests import BaseInvoiceTestCase
+
+
+class TestGetReminderEmailContacts(BaseInvoiceTestCase):
+    def test_get_reminder_email_contacts(self):
+        billing_contacts = ['amanda@example.com', 'bob@example.com']
+        self.account.billingcontactinfo.email_list = billing_contacts
+        self.account.billingcontactinfo.save()
+
+        project_admins = ['carly@example.com', 'dave@example.com']
+        for email in project_admins:
+            user = WebUser.create(self.domain.name, email, 'test123', None, None, email)
```

Comment:
This looks like we aren't cleaning up the users created here. This opens the door for these tests to create unpredictable behavior in other tests, if those tests happen to use these same users. Can we add a cleanup that removes them?